asv_bench/benchmarks/ctors.py

```diff
@@ -55,7 +55,14 @@ class SeriesConstructors:
               [False, True],
               ['float', 'int']]
 
+    # Generators get exhausted on use, so run setup before every call
+    number = 1
+    repeat = (3, 250, 10)
```

Comment:
Why should we care about setting repeat for these? Can't we just use the default `max_time`?

Reply:
The default `repeat` is `(min_n=1, max_n=10, max_time=20.0)`, which assumes `number` can be much greater than `1`. We're not trying to modify the number of samples (`number * repeat`), just how they're collected and setting `repeat` is required to do that.

Reply:
I think I'd still prefer if we just used the default max_time here (assuming that to be bottleneck). Understood it may look weird historically with these but they were wrong anyway

Reply:
IIUC, @qwhelan's approach is preferred since we should get more stable benchmark. With the default `repeat`, we'd observe higher variance in the timings.